docs/quick-start.md

```diff
@@ -20,15 +20,16 @@ First, install and setup cri-resource-manager.
 CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
                jq .tag_name | tr -d '"v'`
 source /etc/os-release
-sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.x86_64.${ID}-${VERSION_ID}.rpm
+[ "$ID" = "sles" -o "$ID" = "opensuse-leap" ] && export ID=suse
+sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
 ```
 
 #### Ubuntu\* and Debian\*
 ```
 CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
                jq .tag_name | tr -d '"v'`
 source /etc/os-release
-pkg=cri-resource-manager_${CRIRM_VERSION}_amd64.${ID}-${VERSION_ID}.deb; curl -LO https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/${pkg}; sudo dpkg -i ${pkg}; rm ${pkg}
+pkg=cri-resource-manager_${CRIRM_VERSION}_${ID}-${VERSION_ID}_amd64.deb; curl -LO https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/${pkg}; sudo dpkg -i ${pkg}; rm ${pkg}
```

Comment:
- Works on Ubuntu 20.04
- Works on Ubuntu 18.04
- Works on Debian 10
- Fails on Debian-sid. VERSION_ID is not set. Would work if VERSION_ID=sid. However, the only variable containing "sid" in /etc/os-release is PRETTY_NAME...
```
debian@e2e-debian-sid-containerd:~$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux bullseye/sid"
NAME="Debian GNU/Linux"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Reply:
@marquiz Any updates planned to this PR to address debian-sid failure reported by @askervin, or do you consider this good to go ?

Reply:
I'm not planning to invest more in this PR. Any further improvements left as a future exercise 😄